Marlin/Configuration.h

```diff
@@ -417,8 +417,11 @@ const bool Z_PROBE_ENDSTOP_INVERTING = false; // set to true to invert the logic
                                  // In RAMPS uses servo pin 2. Can be changed in pins file. For other boards pin definition should be made.
                                  // It is assumed that when logic high = filament available
                                  //                    when logic  low = filament ran out
-//const bool FIL_RUNOUT_INVERTING = true;  // Should be uncommented and true or false should assigned
-//#define ENDSTOPPULLUP_FIL_RUNOUT // Uncomment to use internal pullup for filament runout pins if the sensor is defined.
+#ifdef FILAMENT_RUNOUT_SENSOR
+  const bool FIL_RUNOUT_INVERTING = true;  // Should be uncommented and true or false should assigned
+  #define ENDSTOPPULLUP_FIL_RUNOUT // Uncomment to use internal pullup for filament runout pins if the sensor is defined.
```

Comment:
ah sorry for misunderstanding you. It is the normal config for compile. As it's said at FIL_RUNOUT_INVERTING, you should uncomment it too. That why I put it uncommented, of course I've tried it ;)
![photo226236329031215060](https://cloud.githubusercontent.com/assets/4092862/7428674/e591301a-eff4-11e4-8840-6beb67a956ce.jpg)